### ISSUE TITLE
Distributed code uses ES6 syntax (6.0.0-beta regression)

### ISSUE THREAD

**Issue by andersk** [Grsmto/simplebar#523]
`simplebar/dist/simplebar.{esm.js,js,min.js,umd.js}` and `simplebar-core/dist/simplebar-core.{esm.js,js}` are now built with ES6 syntax, which will be a problem for IE11 users.

```console
$ npm i es-check simplebar@5

+ es-check@5.1.1
+ simplebar@5.2.1
added 102 packages from 47 contributors and audited 102 packages in 4.252s

$ npx es-check --module es5 node_modules/simplebar*/dist/*.js
ES-Check: there were no ES version matching errors!  🎉

$ npm i simplebar@beta

+ simplebar@6.0.0-beta.6
added 4 packages from 5 contributors, updated 1 package and audited 180 packages in 1.688s

$ npx es-check --module es5 node_modules/simplebar*/dist/*.js
ES-Check: there were 6 ES version matching errors.

          ES-Check Error:
          ----
          · erroring file: node_modules/simplebar-core/dist/simplebar-core.esm.js
          · error: SyntaxError: The keyword 'let' is reserved (13:0)
          · see the printed err.stack below for context
          ----

          SyntaxError: The keyword 'let' is reserved (13:0)
[…]

          ES-Check Error:
          ----
          · erroring file: node_modules/simplebar-core/dist/simplebar-core.js
          · error: SyntaxError: The keyword 'let' is reserved (18:2)
          · see the printed err.stack below for context
          ----

          SyntaxError: The keyword 'let' is reserved (18:2)
[…]

          ES-Check Error:
          ----
          · erroring file: node_modules/simplebar/dist/simplebar.esm.js
          · error: SyntaxError: The keyword 'let' is reserved (1235:0)
          · see the printed err.stack below for context
          ----

          SyntaxError: The keyword 'let' is reserved (1235:0)
[…]

          ES-Check Error:
          ----
          · erroring file: node_modules/simplebar/dist/simplebar.js
          · error: SyntaxError: The keyword 'let' is reserved (2023:1)
          · see the printed err.stack below for context
          ----

          SyntaxError: The keyword 'let' is reserved (2023:1)
[…]

          ES-Check Error:
          ----
          · erroring file: node_modules/simplebar/dist/simplebar.min.js
          · error: SyntaxError: The keyword 'let' is reserved (10:14013)
          · see the printed err.stack below for context
          ----

          SyntaxError: The keyword 'let' is reserved (10:14013)
[…]

          ES-Check Error:
          ----
          · erroring file: node_modules/simplebar/dist/simplebar.umd.js
          · error: SyntaxError: The keyword 'let' is reserved (1234:2)
          · see the printed err.stack below for context
          ----

          SyntaxError: The keyword 'let' is reserved (1234:2)
[…]

```

**Comment by andersk** [comment]
Still a problem in `simplebar@6.0.0-beta.7/dist/simplebar.esm.js`, despite bbe26d106b5d9771437e93d767a71e3fc0137568.

[`pkg.module`](https://github.com/rollup/rollup/wiki/pkg.module) should use ES6 modules syntax (which is compiled away by bundlers) but not other ES6 syntax (which is not).

**Comment by Grsmto** [comment]
Thanks again for testing. I'm still working on this actually, the bundling setup with Rollup and Babel is really a nightmare! 🤯

**Comment by andersk** [comment]
Seems mostly fixed in 6.0.0-beta.8 and 6.0.0-beta.9. There’s a `const` in `simplebar-vue` that comes from `vue-runtime-helpers/dist/index.mjs`; see vuejs/rollup-plugin-vue#262.

**Comment by Grsmto** [comment]
I moved the whole codebase to TS which should target ES3.
Everything else should be up to the consumer for polyfilling. Let's see if that works.
